packages/language/test/document.js

```diff
@@ -287,7 +287,7 @@ describe('Document', function() {
         Document.formatTokens_ = util.noop;
       });
 
-      it('should return the language by default', function(done) {
+      it('should return language if no features are requested', function(done) {
```

Comment:
Don't hate me. Isn't language always returned? I was only suggesting we change syntax because it is conditionally returned, while language looked to always be returned.